packages/surveys/src/components/MultipleChoiceMultiQuestion.tsx

```diff
@@ -177,7 +190,6 @@ export default function MultipleChoiceSingleQuestion({
                     value={otherValue}
                     onChange={(e) => {
                       setOtherValue(e.currentTarget.value);
-                      removeItem(otherValue);
```

Comment:
hey @rohan9896 any reason why we've removed this?

Reply:
Hi @ShubhamPalriwala, This was not working anyway, as I checked, bcoz removeItem works when we pass the exact string. I have handled the removal part on submit.

Reply:
removeItem was not working in this case because of batching of state updates in react. Although, removeItem was mentioned before addItem, but bcoz react states doesnt update instantly, inside addItem, we are getting older state which is then getting spread inside along with new value.
<img width="429" alt="Screenshot 2023-10-09 at 3 48 35 PM" src="https://github.com/formbricks/formbricks/assets/56235204/f9383998-9394-4ada-a27b-5d769d4736c9">

Reply:
Have you tested all the corner cases?

Reply:
@ShubhamPalriwala Yes

Reply:
@ShubhamPalriwala I think the issue raised in #1028 will also be solved with this PR.